src/shared/lib/constants/index.ts

```diff
@@ -7,6 +7,9 @@ import { PERMISSIONS } from 'react-native-permissions';
 
 import { Language } from '../types';
 
+const { width: VIEWPORT_WIDTH, height: VIEWPORT_HEIGHT } =
```

Comment:
Will it always correct?  it's evaluated when app just started before any view creation

Reply:
In React Native width and height dimensions can change in two scenarios:
1. The device is rotating and changes the orientation (not our case, we don't support horizontal orientation).
2. The device is foldable. Some android devices can fold but we don't support them for now anyway